### DISCUSSION TITLE
The synchronisation prompts are as follows

### DISCUSSION THREAD

**Discussion by LokChunYu** [floccusaddon/floccus#1526] [General]
The synchronisation prompts are as follows
![iShot_2024-01-15_14 47 57](https://github.com/floccusaddon/floccus/assets/18650603/f3131b7c-4799-4c66-83fe-cb51818f520f)

**Comment by LokChunYu** [comment]
What's going on here?

**Reply by marcelklehr** [reply]
Hi! Can you please create an "issue" if you have problems, this will allow me to help you better
